train.py

```diff
@@ -119,7 +194,13 @@ def __iter__(self):
                         break
             tokenized_inputs = self.tokenizer(buffer, truncation=False)["input_ids"]
             all_token_ids = []
+
+            np_rng = np.random.RandomState(seed=555)
```

Comment:
let's use the `seed` provided in the arguments and pass it as input to the class

Reply:
👍